apps/codelab/src/app/components/angular-routes/angular-routes.component.scss

```diff
@@ -0,0 +1,10 @@
+.container {
```

Comment:
You can also drop container wrapper?

Reply:
Yes. Deleted container and  used  :host instead